br/tests/br_rawkv/run.py

```diff
@@ -73,13 +88,13 @@ def _get_checksum(self, start_key, end_key):
             self._exit_with_error("get checksum failed:\n  start_key: {}\n  end_key: {}", start_key, end_key)
 
 
-    def _run_cmd(self, cmd, *args):
+    def _run_cmd(self, cmd, *args, **env):
         # construct command and arguments
-        cmd_list = [cmd]
+        cmd_list = cmd.split() # `cmd` may contain arguments, so split() to meet requirement of `subprocess.check_output`.
```

Comment:
> `cmd` may contain arguments

it breaks the assumption that `cmd` is used to point to the binary to be executed and `args` represents all the arguments to the command. Is there other method to maintain this assumption?

Reply:
I will try to fix it later.